### DISCUSSION TITLE
Certain keywords cannot be used in the highlights.scm file

### DISCUSSION THREAD

**Discussion by katawful** [tree-sitter/tree-sitter#1148] [Q&A] [answered]
I'm trying to add "break" and "continue" keywords to the highlights file, but tree-sitter complains that these keywords are "invalid node type". This seems standard for other parsers like C, am I missing something with how I write the scheme file?

**Comment by ahlinc** [comment, marked as answer]
I don't see such keywords in your [`grammar.js`](https://github.com/katawful/tree-sitter-obse/blob/development/grammar.js), it seems those aren't implemented yet. You can use in queries only actual things that present in the grammar, cause tree-sitter makes verification for all queries against the grammar and fails if something unknown to it exist among queries.

**Reply by katawful** [reply]
Ok, I see how I can fix this now. Thanks again
